Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1810,6 +1869,12 @@ def tableToMarkdown(name, t, headers=None, headerTransform=None, removeNull=Fals
        :type date_fields: ``list``
        :param date_fields: A list of date fields to format the value to human-readable output.
 
+        :type json_transform: ``TableJsonTransformer``
```

Comment:
what is `TableJsonTransformer`?
the type/description doesn't match the default value set in the code, should be a `header-transformer mapping`

Reply:
Correct, the code changed but forgot to change the docstring. Will fix